cinema/serializers.py

```diff
@@ -45,6 +58,32 @@ class Meta:
         fields = ("id", "title", "description", "duration", "genres", "actors")
 
 
+class TicketSerializer(serializers.ModelSerializer):
+    class Meta:
+        model = Ticket
+        fields = ["id", "row", "seat", "movie_session"]
+        validators = [
+            UniqueTogetherValidator(
+                queryset=Ticket.objects.all(),
+                fields=["movie_session", "row", "seat"]
+            )
+        ]
```

Comment:
I'd recommend to move it to the model. The reason is that if you have multiple serializers for Ticket model, you'll need to add this validator everywhere. But instead you can add this validation in the model class and avoid code duplication.